devstats/queries/issue_activity_since_date.gql

```diff
@@ -1,6 +1,6 @@
 query {
   repository(owner: "_REPO_OWNER_", name: "_REPO_NAME_") {
-    issues(first: 100, filterBy: { since: "2020-01-01T00:00:00Z" }) {
+    issues(first: 100, filterBy: { since: "_START_DATE_" }) {
```

Comment:
Do we know if this will work for the PR query?

Reply:
Hmm I tried adding `pullRequests(first: 100, filterBy: { since: "_START_DATE_" }) {` but then I get `'message': "Field 'pullRequests' doesn't accept argument 'filter'"}]`

Reply:
Thanks for checking!